app/server.go

```diff
@@ -151,6 +151,11 @@ func NewServer(options ...Option) (*Server, error) {
 		s.configStore = configStore
 	}
 
+	// Enable plaintext logger output if this is a "dev" build, but don't override existing settings.
+	if model.BuildNumber == "dev" && s.configStore.FromDefaults() {
```

Comment:
IMO, I think we /should/ override existing settings for developer, so long as we provide an exit hatch from which to recover.

As an alternative to the `FromDefaults`, what if we modify the `Makefile` to conditionally export `MM_LOGSETTINGS_CONSOLEJSON=false`, so that `make run-server` and the like automatically adopt this behaviour? Then developers can export their own `MM_LOGSETTINGS_CONSOLEJSON` if they /really/ want different behaviour, and -- for free -- they'll still get "production" logs when they invoke the binary directly.

Reply:
With the change to the Makefile (see below) I don't think there's a need to track the consoleJSON flag at all, or worry about clobbering config settings.